br/pkg/lightning/config/config.go

```diff
@@ -511,16 +509,15 @@ type FileRouteRule struct {
 }
 
 type TikvImporter struct {
-	Addr               string   `toml:"addr" json:"addr"`
-	Backend            string   `toml:"backend" json:"backend"`
-	OnDuplicate        string   `toml:"on-duplicate" json:"on-duplicate"`
-	MaxKVPairs         int      `toml:"max-kv-pairs" json:"max-kv-pairs"`
-	SendKVPairs        int      `toml:"send-kv-pairs" json:"send-kv-pairs"`
-	RegionSplitSize    ByteSize `toml:"region-split-size" json:"region-split-size"`
-	SortedKVDir        string   `toml:"sorted-kv-dir" json:"sorted-kv-dir"`
-	DiskQuota          ByteSize `toml:"disk-quota" json:"disk-quota"`
-	RangeConcurrency   int      `toml:"range-concurrency" json:"range-concurrency"`
-	DuplicateDetection bool     `toml:"duplicate-detection" json:"duplicate-detection"`
+	Addr             string   `toml:"addr" json:"addr"`
+	Backend          string   `toml:"backend" json:"backend"`
+	OnDuplicate      string   `toml:"on-duplicate" json:"on-duplicate"`
+	MaxKVPairs       int      `toml:"max-kv-pairs" json:"max-kv-pairs"`
+	SendKVPairs      int      `toml:"send-kv-pairs" json:"send-kv-pairs"`
+	RegionSplitSize  ByteSize `toml:"region-split-size" json:"region-split-size"`
+	SortedKVDir      string   `toml:"sorted-kv-dir" json:"sorted-kv-dir"`
+	DiskQuota        ByteSize `toml:"disk-quota" json:"disk-quota"`
+	RangeConcurrency int      `toml:"range-concurrency" json:"range-concurrency"`
 
```

Comment:
let's delete this blank line now that you need to realign the whole block anyway.

Reply:
Done.